app/controllers/admin/customers_controller.rb

```diff
@@ -17,8 +17,9 @@ def index
       respond_to do |format|
         format.html
         format.json do
```

Comment:
It would be really nice to extract this to the API, but I think it would require a lot more changes to be piled on top of this PR...